tests/performance/locustfile.py

```diff
@@ -1,29 +0,0 @@
-import os
```

Comment:
Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?